### ISSUE TITLE
pick_open_reference_otus.py requires HDF5

### ISSUE THREAD

**Issue by wasade** [biocore/qiime#1849]
This was produced from the illumina tutorial. The install of qiime was performed following the QIIME 1.9-rc1 release [notes](https://github.com/biocore/qiime/releases/tag/untagged-8c652ef2fe2a0622066f) (e.g., `pip install https://github.com/biocore/qiime/archive/1.9.0-rc1.tar.gz`)

Relevant versions:

```python
>>> import biom
/usr/local/lib/python2.7/dist-packages/numpy/core/fromnumeric.py:2499: VisibleDeprecationWarning: `rank` is deprecated; use the `ndim` attribute or function instead. To find the rank of a matrix see `numpy.linalg.matrix_rank`.
  VisibleDeprecationWarning)
>>> biom.__version__
'2.1.2'
>>> import qiime
>>> qiime.__version__
'1.9.0-rc1'
>>>
```

Traceback:

```python
Traceback (most recent call last):
  File "/usr/local/bin/pick_open_reference_otus.py", line 450, in <module>
    main()
  File "/usr/local/bin/pick_open_reference_otus.py", line 429, in main
    minimum_failure_threshold=minimum_failure_threshold)
  File "/usr/local/lib/python2.7/dist-packages/qiime/workflow/pick_open_reference_otus.py", line 1076, in pick_subsampled_open_reference_otus
    table = Table.from_hdf5(biom_file)
  File "/usr/local/lib/python2.7/dist-packages/biom/table.py", line 3092, in from_hdf5
    raise RuntimeError("h5py is not in the environment, HDF5 support "
RuntimeError: h5py is not in the environment, HDF5 support is not available
```

config output

```bash
ubuntu@ip-10-150-79-254:10:29:[~/workshop_data/qiime/moving_pictures_tutorial-1.9.0/illumina]$ print_qiime_config.py -t
/usr/local/lib/python2.7/dist-packages/numpy/core/fromnumeric.py:2499: VisibleDeprecationWarning: `rank` is deprecated; use the `ndim` attribute or function instead. To find the rank of a matrix see `numpy.linalg.matrix_rank`.
  VisibleDeprecationWarning)

System information
==================
         Platform:	linux2
   Python version:	2.7.6 (default, Mar 22 2014, 22:59:56)  [GCC 4.8.2]
Python executable:	/usr/bin/python

QIIME default reference information
===================================
For details on what files are used as QIIME's default references, see here:
 https://github.com/biocore/qiime-default-reference/releases/tag/0.1.1

Dependency versions
===================
          QIIME library version:	1.9.0-rc1
           QIIME script version:	1.9.0-rc1
qiime-default-reference version:	0.1.1
                  NumPy version:	1.9.1
                  SciPy version:	0.13.3
                 pandas version:	0.15.2
             matplotlib version:	1.3.1
            biom-format version:	2.1.2
                   h5py version:	Not installed.
                   qcli version:	0.1.1
                   pyqi version:	0.3.2
             scikit-bio version:	0.2.2
                 PyNAST version:	1.2.2
                Emperor version:	0.9.5
                burrito version:	0.9.0
       burrito-fillings version:	Installed.
              sortmerna version:	Not installed.
              sumaclust version:	Not installed.
                          gdata:	Installed.

QIIME config values
===================
For definitions of these settings and to learn how to configure QIIME, see here:
 http://qiime.org/install/qiime_config.html
 http://qiime.org/tutorials/parallel_qiime.html

                     blastmat_dir:	None
      pick_otus_reference_seqs_fp:	/usr/local/lib/python2.7/dist-packages/qiime_default_reference/gg_13_8_otus/rep_set/97_otus.fasta
                         sc_queue:	all.q
      topiaryexplorer_project_dir:	None
     pynast_template_alignment_fp:	/usr/local/lib/python2.7/dist-packages/qiime_default_reference/gg_13_8_otus/rep_set_aligned/85_otus.fasta
                  cluster_jobs_fp:	start_parallel_jobs.py
pynast_template_alignment_blastdb:	None
assign_taxonomy_reference_seqs_fp:	/usr/local/lib/python2.7/dist-packages/qiime_default_reference/gg_13_8_otus/rep_set/97_otus.fasta
                     torque_queue:	friendlyq
                    jobs_to_start:	1
            denoiser_min_per_core:	50
assign_taxonomy_id_to_taxonomy_fp:	/usr/local/lib/python2.7/dist-packages/qiime_default_reference/gg_13_8_otus/taxonomy/97_otu_taxonomy.txt
                         temp_dir:	/tmp/
                     slurm_memory:	None
                      slurm_queue:	None
                      blastall_fp:	blastall
                 seconds_to_sleep:	1

QIIME base install test results
===============================
.........
----------------------------------------------------------------------
Ran 9 tests in 0.050s

OK
```

**Comment by wasade** [comment]
Forgot to link, see #1847

**Comment by gregcaporaso** [comment]
We need to search the code base for ``from_hdf5``, and replace any remaining occurrences with ``load_table``. Thanks for catching this @wasade!

**Comment by mlangill** [comment]
Just came here to post the same bug. Quite annoying since it is one of the last steps in the lengthy open reference picking pipeline.

**Comment by wasade** [comment]
I believe you should be able to pick up where you left off once h5py is
installed though, so the earlier work isn't necessarily lost. But, it will
require a little bit of manual effort and unfortunately this did sneak
through.

**Comment by jairideout** [comment]
Thanks @wasade @mlangill! I'm able to reproduce this locally, will work on fixing this ASAP.

**Comment by jairideout** [comment]
Fix is in #1853.
